vite.config.js

```diff
@@ -4,6 +4,6 @@ import injectHTML from 'vite-plugin-html-inject';
 
 export default defineConfig({
   root: resolve(__dirname, '.'),
-  base: 'goit-markup-team-project/',
+  base: './',
```

Comment:
This change will break github pages.
But, 
`(!) "base" option should start with a slash.`
leading `/` can be added

Reply:
Passed `--base=/goit-markup-team-project/` parameter to Github build action